api/notd/api/response_builder.py

```diff
@@ -103,7 +107,20 @@ async def token_transfer_from_model(self, tokenTransfer: TokenTransfer) -> ApiTo
     async def token_transfers_from_models(self, tokenTransfers: Sequence[TokenTransfer]) -> Sequence[TokenTransfer]:
         return await asyncio.gather(*[self.token_transfer_from_model(tokenTransfer=tokenTransfer) for tokenTransfer in tokenTransfers])
 
-    async def traded_token_from_model(self, tradedToken: TradedToken) -> ApiTradedToken:
+    async def get_collection_statistics(self, collectionStatistics: CollectionStatistics) -> ApiCollectionStatistics:
+        return ApiCollectionStatistics(
+            itemCount=collectionStatistics.itemCount,
+            holderCount=collectionStatistics.holderCount,
+            transferCount=collectionStatistics.transferCount,
+            saleCount=collectionStatistics.saleCount,
+            totalTradeVolume=collectionStatistics.totalTradeVolume,
+            lowestSaleLast24Hours=collectionStatistics.lowestSaleLast24Hours,
+            highestSaleLast24Hours=collectionStatistics.highestSaleLast24Hours,
+            tradeVolume24Hours=collectionStatistics.tradeVolume24Hours,
+        )
+
+
+    async def retrieve_most_traded_token_transfer(self, tradedToken: TradedToken) -> ApiTradedToken:
```

Comment:
this renaming looks weird too. the old function was `traded_token_from_model`, why rename it?